### PR TITLE
[ch3660] Speed init command

### PR DESCRIPTION
**Pull request by B-Galati**
https://app.clubhouse.io/enmarche/story/3660/stocker-l-historique-des-choix-de-notification-des-utilisateurs

### CODE REVIEW

**Comment by B-Galati** [comment]
it's one shot command

File: src/Command/InitializeEmailSubscriptionHistoryCommand.php [hunk 1]
```diff
@@ -55,40 +55,63 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+INSERT INTO adherent_email_subscription_histories (adherent_uuid, subscribed_email_type, action, date) 
+VALUES (:adherent_uuid, :subscribed_email_type, :action, :date)
+SQL;
+        $historyVsTagQuery = <<<'SQL'
+INSERT INTO adherent_email_subscription_history_referent_tag (email_subscription_history_id, referent_tag_id) 
+VALUES (:email_subscription_history_id, :referent_tag_id)
+SQL;
+
+        $connection = $this->em->getConnection();
+        $connection->getConfiguration()->setSQLLogger(null); // Like this we are sure nothing is logged. Memory usage stays stable and it speeds things up.
```
Comment (ottaviano) [inline src/Command/InitializeEmailSubscriptionHistoryCommand.php]: or just `--no-debug` option, not need modify, it's one shot command

File: src/Command/InitializeEmailSubscriptionHistoryCommand.php [hunk 2]
```diff
@@ -55,40 +55,63 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-                $nb = 0;
             }
-        }
 
-        $this->em->flush();
-        $this->em->commit();
+            $progressBar->advance();
+        }
 
+        if ($connection->isTransactionActive()) {
```
Comment (Remg) [inline src/Command/InitializeEmailSubscriptionHistoryCommand.php]: :ok_hand: smooth

File: src/Command/InitializeEmailSubscriptionHistoryCommand.php (hunk 1 above)
Comment (B-Galati) [reply src/Command/InitializeEmailSubscriptionHistoryCommand.php]: Thank you for the info :+1: 

(I will not fix it for this one shot thing)

**Pull request merged**